transactions.js

```diff
@@ -0,0 +1,26 @@
+if (!isLoggedIn()) {
+  location.href = "index.html";
+}
+console.log("You can see this if only you're logged in!");
+
+const userAccountNumberElement = document.getElementById("user-account-number");
+
+const userAccountBalanceElement = document.getElementById(
```

Comment:
Since this element is in the common header section, you should handle in a common place to avoid repetition in other files that will also have this header i.e. Transfer, Withdraw, Deposit, Profile.

Suggested:
Create a new JS file called `common-session.js `. Add this file to the all the session pages (i.e. pages that require user to be logged in before access), in this order:
```
common.js
common-session.js
[current-page-file.js]
```

Every logic that applies to all session pages should be handled in the `common-session.js` file.

Reply:
Done